File: python/quesn1.py
#Name: Yash Agrawal
#Roll: EC20B1059
#Question 1

'''
Write a program which will find all such numbers which are divisible by 7 but are not a multiple of 5, 
between 2000 and 3200 (both included).The numbers obtained should be printed in a comma-separated sequence 
on a single line.
'''
all_numbers=[]
for number in range(2000, 3201):
    if number %7 == 0 and number %5 != 0:
        all_numbers.append(str(number))

print(','.join(all_numbers))

File: c/quesn1.c
#include<stdio.h>
#include<math.h>

int main()
{
    int n,i;
    printf("Displaying the number divisible by 7 and not multiple of 5 between 2000 and 3200\n");
    
    for(n=2002; n<=3200; n=n+7) // 2002 is the first number in the given range which is divisible by 7 and not a multiple of 5
    {
        i=n%5;
        if(i!=0)
        {
          printf("%d,",n);
        }
    }
    
return 0;
}
